libs/core/src/lib/input-group/input-group.component.html

```diff
@@ -28,13 +28,14 @@
                 type="button"
                 fdType="transparent"
                 [compact]="compact"
+                [id]="_addOnButtonId"
                 [attr.tabindex]="buttonFocusable ? 0 : -1"
-                [attr.title]="iconTitle"
+                [attr.title]="iconTitle ? iconTitle : glyph ? glyph : addOnText"
```

Comment:
Here could be used parentheses for bettern readability
`[attr.title]="iconTitle ? iconTitle : (glyph ? glyph : addOnText)"`

Reply:
I had written it that way but prettier formatted it like this 😅  will revert the formatting to make it more readable.